tensorboard/webapp/core/views/layout_container.ts

```diff
@@ -0,0 +1,117 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  OnDestroy,
+} from '@angular/core';
+import {Store} from '@ngrx/store';
+import {fromEvent, Observable, Subject} from 'rxjs';
+import {filter, takeUntil} from 'rxjs/operators';
+
+import {sideBarWidthChanged} from '../actions';
+import {State} from '../state';
+import {getSideBarWidthInPercent} from '../store/core_selectors';
+import {MouseEventButtons} from '../../util/dom';
+
+@Component({
+  selector: 'tb-layout',
```

Comment:
Consider including "resizable" in the name, e.g. "tb-resizable-layout", for more descriptiveness.

Reply:
I am iffy about that. I want this to be used in all pages' layout when possible like https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_dashboard_common/tf-dashboard-layout.ts#L23 and whether it is resizable or not is not the part I want plugin authors to care/know.

Reply:
Hm, then perhaps something along the lines of `tb-dashboard-layout` or `tb-plugin-dashboard-layout`?  With `tb-layout`, it's unclear to me when it should be used.